### PR TITLE
Add format check and clippy to ci-flow

### PR DESCRIPTION
**Pull request by D4nte**
* env CARGO_MAKE_CLIPPY added
* env CARGO_MAKE_CHECK_FORMAT added

### CODE REVIEW

**Comment by sagiegurari** [comment]
thanks.
can you prefix both env vars as 'CARGO_MAKE_RUN_' for example CARGO_MAKE_RUN_CLIPPY?
also do not use the pre hooks, I leave those emtpy so users can override them.
instead just add to the ci-flow dependencies, like bench-ci-flow and examples-ci-flow

**Comment by D4nte** [comment]
Comments implemented.

File: src/Makefile.stable.toml [hunk 1]
```diff
@@ -219,6 +228,10 @@ category = "Documentation"
 
 [tasks.pre-build]
 category = "Build"
+dependencies = [
```
Comment (sagiegurari) [inline src/Makefile.stable.toml]: lets remove this block. no longer needed

Comment (D4nte) [reply src/Makefile.stable.toml]: Done locally but forgot to stage it 😖

**Comment by sagiegurari** [comment]
merged. thanks @D4nte

**Pull request merged**